vignettes/BLCOP.Rmd

```diff
@@ -210,10 +210,10 @@ univariate distributions in a similar way and is used to create the view distrib
 the DAX.
 
 ```{r, echo=TRUE, print=FALSE}
-pick <- matrix(0, ncol = 4, nrow = 1,
-	dimnames = list(NULL, c("SP", "FTSE", "CAC", "DAX")))
-pick[1,"DAX"] <- 1
-viewDist <- list(distribution("unif", min = -0.02, max = 0))
+# pick <- matrix(0, ncol = 4, nrow = 1,
+# 	dimnames = list(NULL, c("SP", "FTSE", "CAC", "DAX")))
+# pick[1,"DAX"] <- 1
+# viewDist <- list(distribution("unif", min = -0.02, max = 0))
```

Comment:
@russey If we're not producing an output, is it worth just adding `eval = FALSE` to the code chunk? I suspect CRAN may not like commetned out code in the vignette